docs/geojson.md

```diff
@@ -1,6 +1,7 @@
 # GeoJson
 
-The `geojson` module contains an implementation of the [GeoJson standard](https://tools.ietf.org/html/rfc7946).
+The `geojson` module contains an implementation of
+the [GeoJson standard](https://tools.ietf.org/html/rfc7946).
```

Comment:
ah my bad, I think in #157 I forgot a prettierrc with `proseWrap` = `always`, will fix